src/server.ts

```diff
@@ -1,23 +1,18 @@
+import dotenv from 'dotenv'
 import express from 'express'
-import { ApolloServer } from 'apollo-server-express'
-import depthLimit from 'graphql-depth-limit'
 import { createServer } from 'http'
 import compression from 'compression'
 import cors from 'cors'
-import resolvers from './resolvers'
-import typeDefs from './typeDefs'
 
-const server = new ApolloServer({
-  typeDefs,
-  resolvers,
-  validationRules: [depthLimit(7)],
-})
+import apolloServer from './apolloServer'
+
+dotenv.config()
 
 const app = express()
 app.use('*', cors())
 app.use(compression())
 
-server.applyMiddleware({ app, path: '/graphql' })
+apolloServer.applyMiddleware({ app, path: '/graphql' })
```

Comment:
💁🏻 we've got both an apollo "server" abstraction an express server that actually binds a network socket and handles HTTP requests, renaming this seemed like it might suggest that a little more clearly

Reply:
Good call 👍